iocore/net/OCSPStapling.cc

```diff
@@ -57,6 +59,142 @@ struct certinfo {
   time_t expire_time;
 };
 
+extern ClassAllocator<FetchSM> FetchSMAllocator;
+
+class HTTPRequest : public Continuation
```

Comment:
This looks like it should be in an anonymous namespace.

Reply:
added anonymous namespace.